docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md

```diff
@@ -52,7 +52,7 @@ You'll need to take a few additional steps to complete the update. In the **head
     ```
 
    > [!NOTE] 
-   > The `/1/` in front of `office.js` in the CDN URL specifies to use the latest incremental release within version 1 of Office.js.
+   > The `/1/` in the `office.js` in the CDN URL specifies to use the latest incremental release within version 1 of Office.js.
```

Comment:
The start of this sentence sounds awkward. Perhaps remove "in the `office.js`" from this sentence, so that it starts like this:

The `/1/` in the CDN URL...